AbstractSQLRules2SQL.ometajs

```diff
@@ -743,6 +744,26 @@ export ometa AbstractSQLRules2SQL {
 			)
 		),
 
+	Case :indent =
+		[	'Case'
+			NestedIndent(indent):nestedIndent
+			[	When(indent)+:whens
```

Comment:
This should be changed to just `When(indent)+:whens` to have the `['Case', ['When' ...` format instead of `['Case', [['When' ...`

Reply:
I thought the same, but when I tried it I got errors. I will try again though.